ui.R

```diff
@@ -151,61 +172,61 @@ dashboardPage(
               "input.howToChooseNBA == 'sel'",
```

Comment:
Be sure to use explicit arguments for all input options. The sliderInput howToChooseNBA is throwing a missing label error.